### DISCUSSION TITLE
Support for Transceivers in WebRTC

### DISCUSSION THREAD

**Discussion by smarfieta** [sipsorcery-org/sipsorcery#817] [Q&A]
Hi

My team is working on live video streaming using WebRTC and currently testing with a simple JS web client.
After getting an error on Firefox, we realised SIP Sorcery does not support Transceivers yet. At the same time I noticed there are some marked as TODO methods in the [IRTCPeerConnection](https://github.com/sipsorcery-org/sipsorcery/blob/28b796200b23b8fcb013033b3b5c4ec305f5e221/src/net/WebRTC/RTCPeerConnection.cs) interface related to Transceivers.

My question to the community: is there someone working on that? 

We would like to sync and thus avoid doing the same things in parallel, but rather join forces. We are willing to chip in and help.

Best,
Marieta

**Comment by sipsorcery** [comment]
> My question to the community: is there someone working on that?

Not as far as I know.

> We would like to sync and thus avoid doing the same things in parallel, but rather join forces. We are willing to chip in and help.

Contributions are always welcome!
